akka-http-core/src/test/scala/akka/http/impl/engine/client/ClientCancellationSpec.scala

```diff
@@ -12,13 +12,14 @@ import scala.concurrent.duration._
 import akka.http.impl.util.AkkaSpecWithMaterializer
 import akka.http.scaladsl.{ ConnectionContext, Http }
 import akka.http.scaladsl.model.{ HttpRequest, HttpResponse }
-import akka.stream.ActorMaterializer
+import akka.stream.SystemMaterializer
 import akka.stream.scaladsl.{ Flow, Sink, Source }
 import akka.stream.testkit.{ TestPublisher, TestSubscriber, Utils }
 import akka.http.scaladsl.model.headers
 
 class ClientCancellationSpec extends AkkaSpecWithMaterializer {
-  val noncheckedMaterializer = ActorMaterializer()
+  // TODO document why this explicit materializer is needed here?
```

Comment:
Seems to be pretty old code, maybe just remove it and the usage and see if something breaks?